SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -771,6 +771,13 @@
                 "td-agent"
             ]
         },
+        {
+            "origin": "Nvidia",
+            "license": "[Nvidia ASL 2.0 License](http://www.apache.org/licenses/LICENSE-2.0)",
```

Comment:
Same comment as above.

Reply:
changed to show they are the same license